packages/webapp/pages/account/security.tsx

```diff
@@ -150,6 +150,9 @@ const AccountSecurityPage = (): ReactElement => {
         }
 
         if (error) {
+          if (error.ui?.messages?.[0].id === 4000001) {
+            displayToast('You cannot unlink this provider');
```

Comment:
Nice fix, I do think it would be best to add this error inside the `KRATOS_ERROR` lib though. *(just for clarity later on)

https://github.com/dailydotdev/apps/blob/7b61f2f74143b9e81561b61769e152ff50670da2/packages/shared/src/lib/kratos.ts#L359